src/rt/util/array.d

```diff
@@ -63,3 +63,9 @@ private void _enforceNoOverlap(in char[] action,
     msg ~= bytes.unsignedToTempString(tmpBuff, 10);
     throw new Error(msg);
 }
+
+private const(void)* arrayToPtr(const void[] array) @trusted
+{
+    // Ok because the user will never dereference the pointer
```

Comment:
Hm... probably should return ptrdiff_t instead of size_t, since pointer arithmetic is signed.